core/src/test/java/io/substrait/relation/VirtualTableScanTest.java

```diff
@@ -0,0 +1,52 @@
+package io.substrait.relation;
+
+import static io.substrait.expression.ExpressionCreator.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import io.substrait.expression.Expression;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class VirtualTableScanTest {
+
+  @Test
+  void check() {
+    VirtualTableScan virtualTableScan =
+        ImmutableVirtualTableScan.builder()
+            .addDfsNames(
+                "string",
+                "struct",
+                "struct_field1",
+                "struct_field2",
+                "list",
+                "list_struct_field1",
+                "map",
+                "map_key_struct_field1",
+                "map_value_struct_field1")
+            .addRows(
+                struct(
+                    false,
```

Comment:
If I'm understanding the naming correctly, doesn't this boolean correspond to the name "string", and then the string below it corresponds to the name "struct"?